misc/python/materialize/checks/all_checks/create_table.py

```diff
@@ -60,9 +60,6 @@ def validate(self) -> Testdrive:
                 > SELECT * FROM create_table_view2;
                 2
 
-                > SHOW CREATE TABLE create_table1;
-                materialize.public.create_table1 "CREATE TABLE \\"materialize\\".\\"public\\".\\"create_table1\\" (\\"f1\\" \\"pg_catalog\\".\\"int4\\", \\"f2\\" \\"pg_catalog\\".\\"int4\\" NOT NULL DEFAULT 1234)"
```

Comment:
There are a bunch of deleted SHOW CREATE statements here and in other test files. Are those not useful anymore?